tools/nanopolish/nanopolish_eventalign.xml

```diff
@@ -0,0 +1,111 @@
+        <param type="data" name="input_reads_bam" format="bam" label="Reads aligned to the reference genome" help="(-b)" />
+        <param type="data" name="input_genome_fa" format="fasta" label="The reference genome" help="(-g)"/>
+
+        <section name="adv" title="Optional data inputs">
+            <!-- optional inputs -->
+            <param type="data" name="input_models_fofn" format="txt" optional="true" label="Read alternative k-mer models" help="(--models-fofn)" />
+       </section>
+
+        <!-- optional params -->
+        <param name="window" type="text" optional="true" label="find variants in window chromsome_name:start-end" help="(-w)"/>
+
+        <!-- optional flags -->
+        <param name="verbose" type="boolean" truevalue="--verbose" falsevalue="" checked="false" label="verbose output" help="(-v)" />
+        <param name="summary" type="boolean" truevalue="--summary" falsevalue="" checked="true" label="Summarize the alignment of each read/strand" help="(--summary)" />
+        <param name="samples" type="boolean" truevalue="--samples" falsevalue="" checked="false" label="Write the raw samples for the event to the tsv output" help="(--samples)" />
+        <param name="scale_events" type="boolean" truevalue="--scale-events" falsevalue="" checked="false" label="Scale events to the model, rather than vice-versa" help="(--scale-events)" />
+        <param name="SAM" type="boolean" truevalue="--sam" falsevalue="" checked="false" label="write output in SAM format" help="(--sam)" />
+        <param name="print_read_names" type="boolean" truevalue="--print-read-names" falsevalue="" checked="false" label="Print read names instead of indexes" help="(--print-read-names)" />
+
+    </inputs>
+
+    <outputs>
+      <!-- variants consensus outputs -->
+        <data name="output_summary" format="txt" from_work_dir="eventalign-summary.txt" label="eventalign summary of reads/strands" />
+        <data name="output_eventalign" format="txt" from_work_dir="eventalign.out" label="Computed variants"/>
+    </outputs>
+    <tests>
+        <test>
+      <!-- index test -->
+            <param name="input_merged" ftype="fasta" value="reads.fasta" />
+            <param name="input_reads_raw" ftype="fast5.tar.gz" value="fast5_files.tar.gz" />
+            
+      <!-- variants consensus test -->
+            <param name="input_reads_bam" value="reads.sorted.bam" />
+            <param name="input_genome_fa" value="draft.fa" />
+            <param name="window" value="tig00000001:200000-200010" />
+            <param name="SAM" value="true" />
+
+            <output name="output_summary" file="eventalign-summary.txt" />
+            <output name="output_eventalign" file="reads-draft.eventalign.sam"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        Usage: nanopolish eventalign [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa
+        Align nanopore events to reference k-mers
```

Comment:
Listing the commandline here again is less useful, a nice description, tipps or links to further documentation is more useful.